.environment.yml

```diff
@@ -11,10 +11,12 @@ dependencies:
   - pip
   - pytest
   - python=3.6
-  - pytorch=0.4
+  - pytorch=1.0.1
   - scipy
   - tifffile
   - torchvision
   - tqdm
+  - scikit-image
   - pip:
     - czifile==0.1.5
+    - t4
```

Comment:
You can split up package dependencies in `setup.py` now!
Example from [aics-deformation `setup.py`](https://github.com/AllenCellModeling/aics-deformation/blob/master/setup.py#L31)